cookiecutter.json

```
@@ -1,7 +1,7 @@
 {
   "author_given_names": "Eva Lu",
   "author_family_names": "Ator",
-  "author_email": "eva.lu.ator@ucl.ac.uk",
+  "author_email": "{{'.'.join(cookiecutter.author_given_names.lower().split() + cookiecutter.author_family_names.lower().split())}}@ucl.ac.uk",
```

```suggestion
  "author_email": "{{'.'.join(cookiecutter.author_given_names.lower().split() + '.'.join(cookiecutter.author_family_names.lower().split()))}}@ucl.ac.uk",
```
In case there are multiple family names (the variable name implies that there could be?)

---

The current implementation already handles this as the `join` call is on the concatenation of the lists of given and family names generated by the `split` calls, but it is difficult to spot this as having the code embedded in a string makes for difficult reading! For example `author_given_names` = `"Eval Lu"` and `author_family_names` = `"At Or"` will give suggested default `author_email` = `"eva.lu.at.or@ucl.ac.uk"` currently.